README.md

```diff
@@ -26,8 +26,13 @@ Install the dependencies using
 ```
 npm install
 ```
+You will have to create a Genius API Access Token that you can get from the link below: https://docs.genius.com/
 
-Download the models using the drive link below and place them in server/models
+After obtaining the key, we have to add it to server/.env file with ACCESS_TOKEN header, like so
+![image](https://user-images.githubusercontent.com/34500350/115106691-e23b4780-9f83-11eb-8297-0a66ed81cbe5.png)
```

Comment:
can be a written on the readme instead of having an image
```
API_TOKEN=<INSERT TOKEN HERE>
```

anyhow this also works